cli/types/index.d.ts

```diff
@@ -4,7 +4,7 @@
 //                 Mike Woudenberg <https://github.com/mikewoudenberg>
 //                 Robbert van Markus <https://github.com/rvanmarkus>
 //                 Nicholas Boll <https://github.com/nicholasboll>
-// TypeScript Version: 2.9
+// TypeScript Version: 3.4

```

Comment:
This seems like a spooky/major change? Are we sure this isn't going to impact folks downstream maybe using earlier TS?

Reply:
It's needed to support `globalThis`. It may well affect your users, so you might want to put this PR on hold until more of your users are on 3.4 or above.

Reply:
We've run into this before, so this is not the first time this has been suggested. https://github.com/cypress-io/cypress/pull/4950#pullrequestreview-342540916

Reply:
Since we _just_ accepted a pull to move to 3.0, I'm guessing that's going to be a conflict -- curious how much of this works at that level?

Reply:
3.4 is the minimum version if we want to use globalThis